uint8 LEFT_HAND=0
uint8 RIGHT_HAND=1
uint8 LEFT_ARM=2
uint8 RIGHT_ARM=3
uint8 TORSO=4
uint8 group_id
float64[] position
---
uint8 SUCCEEDED=0
uint8 FAILED=1
uint8 result_status